VTEX - Catalog API Seller Portal.json

```
@@ -0,0 +1,4595 @@
+                                        "id": {
+                                            "type": "string",
+                                            "title": "id",
+                                            "description": "Brand unique identifier number.",
+                                            "example": "863"
+                                        },
+                                        "name": {
+                                            "type": "string",
+                                            "title": "name",
+                                            "description": "Brand Name.",
+                                            "example": "Zwilling"
+                                        },
+                                        "isActive": {
+                                            "type": "boolean",
+                                            "title": "isActive",
+                                            "description": "The condition defines if the brand is active or inactive.",
+                                            "example": false
+                                        },
+                                        "createdAt": {
+                                            "type": "string",
+                                            "title": "createdAt",
+                                            "description": "Date when the brand was created.",
+                                            "example": "2021-01-18T14:41:45.696488+00:00"
+                                        },
+                                        "updatedAt": {
+                                            "type": "string",
+                                            "title": "updatedAt",
+                                            "description": "Last date when the brand was updated.",
+                                            "example": "2021-01-18T14:41:45.696488+00:00"
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+            "put": {
+                "tags": [
+                    "Brand"
+                ],
+                "summary": "Update Brand",
+                "description": " >📘 This API is part of the [Seller Portal Catalog](https://help.vtex.com/en/tutorial/how-the-seller-portal-catalog-works--7pMB6YOt6YQDQQbzFB4Pxp). This functionality is in the Beta stage and can be discontinued at any moment at VTEX's discretion. VTEX will not be responsible for any instabilities caused by its use or discontinuity. If you have any questions, please contact [our Support Center](https://support.vtex.com/hc/en-us/requests). \r\n\r\n The endpoint updates an existing brand. \r\n\r\n## Request body example\r\n\r\n```json\r\n{\r\n  \"name\": \"Zwilling\",\r\n  \"isActive\": true\r\n}\r\n```",
```

```suggestion
                "description": " >📘 This API is part of the [Seller Portal Catalog](https://help.vtex.com/en/tutorial/how-the-seller-portal-catalog-works--7pMB6YOt6YQDQQbzFB4Pxp). This functionality is in the Beta stage and can be discontinued at any moment at VTEX's discretion. VTEX will not be responsible for any instabilities caused by its use or discontinuity. If you have any questions, please contact [our Support Center](https://support.vtex.com/hc/en-us/requests). \r\n\r\n Updates an existing brand. \r\n\r\n## Request body example\r\n\r\n```json\r\n{\r\n  \"name\": \"Zwilling\",\r\n  \"isActive\": true\r\n}\r\n```",
```